### PR TITLE
Docs changes

### PR DESCRIPTION
**Pull request by sanjanag**
Addresses these comments
https://cleanlabinc.slack.com/archives/C04DC8K716X/p1678837871858979

### CODE REVIEW

File: src/cleanvision/imagelab.py [hunk 1]
```diff
@@ -589,6 +589,7 @@ def visualize(
 
     # Todo: Improve mypy dict typechecking so this does not return any
     def get_stats(self) -> Any:
+        """Returns statistics on images like brightness, color space, aspect ratio"""
```
Comment (jwmueller) [inline src/cleanvision/imagelab.py:592]:
```suggestion
        """Returns statistics computed about images when auditing the data such as: brightness, color space, aspect ratio, etc."""
```

**Review by jwmueller — APPROVED** [review]
just had one suggestion

File: src/cleanvision/imagelab.py (hunk 1 above)
Comment (jwmueller) [reply src/cleanvision/imagelab.py:592]: say "dict of statistics" instead of "statistics" if this indeed returns a dict

Comment (jwmueller) [reply src/cleanvision/imagelab.py:592]: In future would be good to clarify when methods return `None`, I assume this one does if the stats haven't been computed yet

Comment (sanjanag) [reply src/cleanvision/imagelab.py:592]: Yes, this can return None.

**Review by ulya-tkch — APPROVED** [review]
LGTM!

File: src/cleanvision/imagelab.py (hunk 1 above)
Comment (jwmueller) [reply src/cleanvision/imagelab.py]: I was saying you should clarify this in the docstring. Adding it myself now.

**Pull request merged**